Mac target

bgulanowski opened the pull request:
- Xcode 9.4 bookkeeping and changes
- new target
- make the MBEMetalView support Mac as well as iOS (using conditional compilation)
- Add `Config.xcconfig` as a means to set team without changing project file

I choose to reuse the existing view class to achieve:
- avoid code duplication
- avoid factoring out a shared renderer object (done in later chapters)
- highlight differences between Mac and iOS layer handling

It's been ten years since I tried to mess with layers on macOS. The docs could be a bit more clear on the canonical approach to using a custom layer in an NSView. Specifically, the docs suggest that `-updateLayer` will be called, but this is dependent on various factors. It wasn't called for me.

`-displayLayer:`works, but may not be the ideal design.

internaut commented:
Uh, never mind. Looks like this is done automatically when the texture is released.

bgulanowski committed:
Xcode 9.4 bookkeeping

bgulanowski committed:
Xcode 9.4 suggested target settings.

bgulanowski committed:
Add macOS target.

bgulanowski committed:
Xcode 9.4 changes and bookkeeping.

bgulanowski committed:
Use config file to set development team.

Changes to this file will no longer be tracked by git.

bgulanowski committed:
Stop tracking config file.

This file will be used by individuals to avoid polluting the repo with specific team settings.

bgulanowski commented:
Renamed branch. Re-opening.
